README.md

```diff
@@ -47,245 +17,134 @@ After you're finished please remove all the comments and instructions!
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
+<!-- DESCRIPTION -->
+## 📖 About the Project <a name="#about-project"></a>
 
-# 📖 [your_project_name] <a name="about-project"></a>
+In the upcoming projects, you will use a relational database to create the data structure for a vet clinic. You will start with one table, and step by step, you will create a complete database with data about:
 
-> Describe your project in 1 or 2 sentences.
+    animals
+    animals' owners
+    clinic employees
+    visits
 
-**[your_project__name]** is a...
+Apart from building data structure, you will be challenged with tasks that require SQL queries to answer specific questions.
```

Comment:
- [ ] [OPTIONAL] Instead of just copying the project requirements I'd suggest you provide a proper description of the project please keep in mind recruiters and future clients will look through the readme before opening your project and It's best to catch their attention and keep them engaged as much as you can. 😉

Reply:
I fixed the description section for another one according to the project and future features.

![FIX-02](https://github.com/jlcarrascof/Vet-clinic-database/assets/86327867/3ff89839-c51e-47fb-a7e4-1f69322f3bfe)